Source/DateTimePicker.swift

```diff
@@ -60,9 +62,25 @@ public protocol DateTimePickerDelegate: class {
             doneButton.backgroundColor = doneBackgroundColor
         }
     }
+    /// custom Background Color Content Viewr, default to white
+    public var contentViewBackgroundColor: UIColor? {
```

Comment:
Please set `.white` as default value here just so you don't have to use the ternary operator on line 319

Reply:
I agree with you!